mosip/on-prem/README.md

```diff
@@ -43,7 +43,21 @@ ansible-playbook -i hosts.ini ports.yaml
 ```
 ansible-playbook -i hosts.ini swap.yaml
 ```
+## Localhost Mapping
+Below contains some of the information about localhost mapping in brief:
 
+we faced an issue with nodal instances where we were getting issues as localhost mapping was not present in hosts file of the system.
```

Comment:
An issue with the nodal instances was observed as localhost mapping was not present in the hosts file of the system.